desktop/release-notes.md

```diff
@@ -24,12 +24,69 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.13.1
+2022--
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### Upgrades
+
+- [Docker Compose v2.12.1](https://github.com/docker/compose/releases/tag/v2.12.1)
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Fixed a possible segfault when interrupting `docker pull` with `Control+C` or `CMD+C`.
+- Increased the default DHCP lease time to avoid the VM's network glitching and dropping connections every two hours.
+- Enabled [contextIsolation](https://www.electronjs.org/docs/latest/tutorial/context-isolation) and [sandbox](https://www.electronjs.org/docs/latest/tutorial/sandbox) mode inside Docker Extensions. Now Docker Extensions run in a separate context and this limits the harm that malicious code can cause by limiting access to most system resources.
```

Comment:
FYI : Waiting on this to pass some tests to add it to the release, if not we will remove this entry.